commands/src/main/java/org/wildfly/extras/creaper/commands/elytron/AddExtensionAndSubsystem.java

```diff
@@ -0,0 +1,23 @@
+package org.wildfly.extras.creaper.commands.elytron;
+
+import org.wildfly.extras.creaper.core.ServerVersion;
+import org.wildfly.extras.creaper.core.online.OnlineCommand;
+import org.wildfly.extras.creaper.core.online.OnlineCommandContext;
+import org.wildfly.extras.creaper.core.online.operations.Address;
+import org.wildfly.extras.creaper.core.online.operations.Operations;
+import org.wildfly.extras.creaper.core.online.operations.admin.Administration;
+
+public final class AddExtensionAndSubsystem implements OnlineCommand {
```

Comment:
Boooo! This should have been called `AddElytron...`.

Reply:
+1, thanks, I will fix it; however I think whether to use `AddElytronExtensionAndSubsystem` or just `AddElytron`. WDYT?

Reply:
`AddElytron` sounds good to me.